### PR TITLE
Fix invalid CSS rules

### PR DESCRIPTION
**Pull request by whymarrh**
This PR fixes/removes a few invalid CSS rules.

### CODE REVIEW

**Review by rekmarks — APPROVED** [review]
LGTM!

File: ui/app/css/variables/colors.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // These are the colors of the MetaMask design system
-// Only design system colors should be added, no superfulous variables
+// Only design system colors should be added, no superfluous variables
```
Comment (brad-decker) [inline ui/app/css/variables/colors.scss:2]: *checks blame* I find this change to be Superfluous.

**Review by brad-decker — APPROVED** [review]

File: ui/app/css/variables/colors.scss (hunk 1 above)
Comment (whymarrh) [reply ui/app/css/variables/colors.scss:2]: This should've been in #9038 😅

Comment (brad-decker) [reply ui/app/css/variables/colors.scss:2]: lol, no worries. I'm just mildly concerned that the vscode plugin that prevents me from contributing a wide array of typos has failed.

**Pull request merged**